front/src/applications/stdcmV2/views/StdcmViewV2.tsx

```diff
@@ -5,60 +5,138 @@ import { useTranslation } from 'react-i18next';
 import { useSelector } from 'react-redux';
 
 import useStdcm from 'applications/stdcm/hooks/useStdcm';
-import { useOsrdConfSelectors } from 'common/osrdContext';
+import { useOsrdConfSelectors, useOsrdConfActions } from 'common/osrdContext';
+import type { StdcmConfSliceActions } from 'reducers/osrdconf/stdcmConf';
+import { useAppDispatch } from 'store';
 
 import StdcmConfig from '../components/StdcmConfig';
 import StdcmHeader from '../components/StdcmHeader';
 import StdcmResults from '../components/StdcmResults';
-import type { StdcmSimulationResult } from '../types';
+import type { StdcmSimulation, StdcmSimulationInputs } from '../types';
 
 const StdcmViewV2 = () => {
   const { t } = useTranslation('stdcm');
-  const { launchStdcmRequest, cancelStdcmRequest, isPending, stdcmV2Results, pathProperties } =
-    useStdcm();
+  const {
+    launchStdcmRequest,
+    cancelStdcmRequest,
+    isPending,
+    isSuccessful,
+    isStdcmResultsEmpty,
+    stdcmV2Results,
+    pathProperties,
+  } = useStdcm();
   const { getScenarioID } = useOsrdConfSelectors();
   const scenarioID = useSelector(getScenarioID);
 
   const [currentSimulationInputs, setCurrentSimulationInputs] = useState<
-    StdcmSimulationResult['input'] | undefined
+    StdcmSimulationInputs | undefined
   >(undefined);
 
-  const [interactedResultsElements, setInteractedResultsElements] = useState(false);
+  const [simulationsList, setSimulationsList] = useState<StdcmSimulation[]>([]);
+  const [selectedSimulationIndex, setSelectedSimulationIndex] = useState<number>(-1);
+  const [showSuccessBanner, setShowSuccessBanner] = useState<boolean>(false);
+  const [retainedSimulationIndex, setRetainedSimulationIndex] = useState(-1);
+  const dispatch = useAppDispatch();
+  const { resetStdcmConfig } = useOsrdConfActions() as StdcmConfSliceActions;
 
+  const selectedSimulation = simulationsList[selectedSimulationIndex];
+
+  // hide the simulation available card when the user interacts with the outputs.
+  const onInteractWithOutputs = () => setShowSuccessBanner(false);
+
+  // hide the simulation available card when the user interacts with the inputs.
+  // Even if currentSimulationInputs changed each times inputs changed, we can't use it to hide the card.
+  // Because it can change when the user change the selected simulation too.
+  const onInteractWithInputs = () => setShowSuccessBanner(false);
```

Comment:
After a discussion with @thibautsailly, we noticed that we want to hide the success banner as soon as the user has a click interaction anywhere in the page. So it will save you a lot of prop drilling with these functions :)

Reply:
Done